composer.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "ezsystems/ezstudio-installer",
+  "description": "Installer for eZ Studio",
+  "license": "TTL-2.0",
+  "authors": [
+    {
+      "name": "eZ Studio dev-team",
+      "homepage": "https://github.com/ezsystems/ezstudio-installer/graphs/contributors"
+    }
+  ],
+  "require": {
+    "ezsystems/ezpublish-kernel": "^6.2"
+  },
+  "autoload": {
+    "psr-4": {
+      "EzSystems\\StudioInstallerBundle\\": ""
+    }
+  },
+  "extra": {
+    "branch-alias": {
+      "dev-master": "1.0.x-dev"
```

Comment:
Are you sure we want to tag a 1.x right away ? it does imply that we will maintain BC from that point. 0.x doesn't have that constraint as per semver.